app/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,8 @@
   xmlns:tools="http://schemas.android.com/tools"
   android:installLocation="preferExternal">
 
+  <uses-feature android:name="android.hardware.camera" android:required="false" />
```

Comment:
If camera hardware is optional, are we going to check the existence of it during runtime?

Reply:
Yeah, it's going to be optional since we also will add the capability to type in a card number.

You have a good point about checking for the existence of a camera though. I'll add this check.

Reply:
Can I do this in the next PR in the Simple ID feature?

Reply:
Discussed offline: Will be done in next PR.